src/features/contributor-activity/ContributorActivityFeature.tsx

```
@@ -0,0 +1,198 @@
+        <>
+          <div className="feature-header">
+            <h1>Contributor Activity Matrix</h1>
+            <p>
+              Tracking activity for <strong>{organization}</strong> since {fromDate}
+              {!tokenProvided && <span className="rate-limit-warning"> (Add token for more data)</span>}
+              {cacheCleared && <span className="cache-cleared"> Cache cleared!</span>}
+            </p>
+          </div>
+          
+          <div className="header-actions">
+            <button onClick={handleClearCache} className="btn-clear-cache">
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)